### ISSUE TITLE
Remote java debug - not binding to localhost

### ISSUE THREAD

**Issue by luisinserra** [codenvy/codenvy#1799]
Hi,

I have a java maven tomcat project.

in terminal I start tomcat in debug mode this way:

in /home/user/tomcat8/bin
sh catalina.sh jpda run
tomcat starts and deploy projects ok

then I enter in menu: run -> Edit Debug

I leave options unchanged, server localhost and port 8000 (tomcat debug)
Then I click Debug.
There's no message connecting to localhost.
Debug cannot connect with localhost.
It used to work this way before.

Code runs ok and does not stop in breakpoints.

Does we have something changed in debug process?

**Comment by eivantsov** [comment]
@luisinserra is this on codenvy.io?

**Comment by luisinserra** [comment]
Yes

**Comment by eivantsov** [comment]
Just checked it:

![image](https://cloud.githubusercontent.com/assets/5337267/23070878/2b337adc-f535-11e6-8015-dd04d3866e57.png)

**Comment by eivantsov** [comment]
What happens when you click debug? Error? Is there a notification?

Trying it in an incognito mode is a good idea.

**Comment by luisinserra** [comment]
Ohhhhh! It is working now.
Yesterday I tryed several times without success.
Now it is working.

I don't know if something changed, anyway it's ok now.
Thanks.

I am closing this issue.
Thanks.

**Comment by luisinserra** [comment]
Hi! I'm here again.
The issue I related is happening again.
After starting tomcat in debug mode (sh catalina.sh jpda run), I cannot connect to localhost port 8000

Run -> Edit Debug configurations
Remote java
Connect to process on workspace machine checked
port 8000

Clicking Debug nothing happens.

No msg like "connecting to localhost 8000".
Nothing.

Can I create a custom command to connect debuger to localhost?
I think the problem is not with tomcat. It is started and listening to port 8000
The last time you answered, I tryed and it worked fine, After that, never more.

Any idea about not connecting?

Thanks
